dev/ci/ci-stalmarck.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+set -e
+
+ci_dir="$(dirname "$0")"
+. "${ci_dir}/ci-common.sh"
+
+git_download stalmarck
+
+if [ "$DOWNLOAD_ONLY" ]; then exit 0; fi
+
+( cd "${CI_BUILD_DIR}/stalmarck"
+  make
+  make install
+
+  dune build @install
```

Comment:
Going by the mathcomp_word script maybe
```suggestion
  dune build @install -p coq-stalmarck-tactic
```
would work?

Reply:
No, this never worked for me locally outside Coq's CI. This triggers Dune issue https://github.com/ocaml/dune/issues/4814
```shell
Error: The following <package>.install are missing:
- _build/default/coq-stalmarck.install
```

Reply:
@SkySkimmer could you try the following locally? It's the only hack workaround I can think that doesn't require me to purge useful files from the repo:

```shell
  make
  make install

  rm coq-stalmarck.opam
  dune build @install -p coq-stalmarck-tactic
  dune install -p coq-stalmarck-tactic --prefix="$CI_INSTALL_DIR"
```

Reply:
That seems to work
NB `rm -f` is better so that the script can be rerun without failing

Reply:
Done in 2728389bb21, do you need to do the "full CI" thing again?

Reply:
I had the magic label on ;)